### ISSUE TITLE
-cd namespace is not exported by release manager

### ISSUE THREAD

**Issue by clemensutschig** [opendevstack/ods-jenkins-shared-library#828]
**Is your feature request related to a problem? Please describe.**
Suppose my integration tests need a secret or alike out of the -cd namespace - and for some reason the namespace vanishes in a catastrophic failure - then we have no way to recover it based on bitbucket data.

**Describe the solution you'd like**
as part of the RM release build, -cd namespace exported, so that one can rebuild it from template.ymls

**Comment by clemensutschig** [comment]
@metmajer fyi as discussed

**Comment by clemensutschig** [comment]
I have an idea how it could work, but this work is blocked by a tailor bug that needs to make it to on premise first :)

**Comment by michaelsauter** [comment]
@clemensutschig I think you are referring to the bug you opened yesterday, right? The one regarding exluded resource kinds? That made it into 1.3.4, which is not the version in the linked PR. I need to open a new PR for that new version.

On a side note, personally, I do not recommend to use the export feature more and more. I believe the best approach in general (though I know it does not fit neatly into the current architecture) would be to define the K8s resources upfront in code and evolve this as the CD namespace evolves. This is the direction that ODS pipeline takes (e.g. see the [user installation](https://github.com/opendevstack/ods-pipeline/blob/master/docs/user-installation.adoc)), which is also due to the fact that ODS pipeline only supports Helm, which doesn't have export functionality. Tailor has, but it is very tricky to get "right", and the Tailor functionality has lots of edge cases that luckily do not appear too often, but they are there and will cause issues from time to time. Just throwing in my 2c ;)

**Comment by clemensutschig** [comment]
and you are totally right @michaelsauter  - but I want to be rather save than sorry ... (at least for the time being).

**Comment by michaelsauter** [comment]
@clemensutschig Update done in master and 4.x.

**Comment by clemensutschig** [comment]
@s2oBCN  / @albertpuente  can we get this version(s) in-house on the dev cluster ... I can work with them?

**Comment by albertpuente** [comment]
If you mean the dev cluster which uses bitbucket-dev, it is updated automatically every hour with the contents from github.

**Comment by clemensutschig** [comment]
the images as well?

**Comment by albertpuente** [comment]
No, those need to be updated manually. Will do.
